pandas/tests/internals/test_internals.py

```diff
@@ -1432,3 +1432,11 @@ def test_make_block_no_pandas_array(block_maker):
         )
         assert result.dtype.kind in ["i", "u"]
         assert result.is_extension is False
+
+
+def test_concat_value():
+    data = [{"A": None}]
```

Comment:
Could you move this test to pandas/tests/reshape/concat/test_concat.py

and also add the Github issue number as a comment?

Reply:
done